perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ResourcesManagerMethod.java

```diff
@@ -1419,5 +1419,55 @@ public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
 			return ac.getResourcesManager().getGroupAssignments(ac.getSession(),
 				ac.getResourceById(parms.readInt("resource")), attrNames);
 		}
+	},
+
+	/*#
+	 * Try to activate the group-resource status. If the async is set to false, the validation is performed
+	 * synchronously. The assignment will be either ACTIVE, in case of a successful synchronous call, or it will be
+	 * PROCESSING in case of an asynchronous call. After the async validation, the state can be either ACTIVE or
+	 * FAILED.
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param resource int Resource <code>id</code>
+	 * @param async boolean if true the validation is performed asynchronously
```

Comment:
I think we could specify here the default value.

Reply:
Added.